plugin/evm/vm_upgrade_bytes_test.go

```diff
@@ -146,6 +146,9 @@ func TestVMUpgradeBytesPrecompile(t *testing.T) {
 }
 
 func TestVMUpgradeBytesNetworkUpgrades(t *testing.T) {
+	// Skipping the NetworkUpgrade Tests until we get a new NetworkUpgrade.
+	// The existing and only NetworkUpgrade that exists is SubnetEVM Upgrade which is now required at genesis.
+	t.Skip("no non-required network upgrades to test network upgrade abort")
```

Comment:
could we re-enable these tests using the new config flag override?

Reply:
done